root/layout.tt

```diff
@@ -71,6 +71,17 @@
             </div>
         [%- END -%]
 
+        [%- IF new_edit_notes && new_edit_notes_mtime >
+                (c.req.cookies.new_edit_notes_dismissed_mtime.value || 0) -%]
+            <div class="banner new-edit-notes">
+                <p>
+                    [% l('Notes have been left on some of your edits. Please {uri|click here} to read them.',
```

Comment:
`... edits. Read {link|your edits with new notes} now.` or something?

Background: http://www.w3.org/QA/Tips/noClickHere, http://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

(Regarding `uri`, not all translators might understand it.)